js/src/modals/Verification/store.js

```diff
@@ -91,27 +101,28 @@ export default class VerificationStore {
         this.error = 'Failed to fetch the fee: ' + err.message;
       });
 
-    const isVerified = checkIfVerified(contract, account)
-      .then((isVerified) => {
-        this.isVerified = isVerified;
+    const accountIsVerified = checkIfVerified(contract, account)
+      .then((accountIsVerified) => {
+        this.accountIsVerified = accountIsVerified;
```

Comment:
Will mobx recognise `load` as an async action if we return the `Promise.all` below?